FBTweak/FBTweakCategory.h

```diff
@@ -12,29 +12,52 @@
 @class FBTweakCollection;
 
 /**
-  @abstract A named grouping of collections.
+ @abstract Represents an named array of \c FBTweakCollection objects.
  */
-@interface FBTweakCategory : NSObject <NSCoding>
+@protocol FBTweakCategory <NSObject>
 
 /**
-  @abstract Creates a tweak category.
-  @discussion This is the designated initializer.
+ @abstract The name of the category.
  */
-- (instancetype)initWithName:(NSString *)name;
+@property (nonatomic, copy, readonly) NSString *name;
 
 /**
-  @abstract The name of the category.
+ @abstract The collections contained in this category.
  */
-@property (nonatomic, copy, readonly) NSString *name;
+@property (nonatomic, copy, readonly) NSArray<FBTweakCollection *> *tweakCollections;
+
+/**
+ @abstract Resets all tweaks in \c tweakCollection to their default value.
+ */
+- (void)reset;
+
+@optional
+
+/**
+ @abstract Asynchronously updates \c tweakCollections to the latest value, and calls \c completion
+ when done. Upon error \c error is set, otherwise \c error is \c nil.
+ @param completion Completion block to be called when the update is complete. \c completion can be
+ called after this method returns.
+ @discussion When the update operation is complete, is it expected that the \c tweakCollection will
+ be update with the latest object.
+ */
+- (void)updateWithCompletion:(void (^)(NSError *error))completion;
+
+@end
 
 /**
-  @abstract The collections contained in this category.
+ @abstract A named grouping of collections.
  */
-@property (nonatomic, copy, readonly) NSArray *tweakCollections;
+@interface FBNativeTweakCategory : NSObject <FBTweakCategory, NSCoding>
```

Comment:
I think there are better names that matches the description `A named grouping of collections` than `FBNativeTweakCategory`.
Perhaps `FBGroupedTeakCategory` or something similar.